### ISSUE TITLE
Add a `.go` implementation of Burst Sort (radix-sort variant)

### ISSUE THREAD

**Issue by IAmRiteshKoushik** [Ashrockzzz2003/Data_Structures_and_Algorithms#38]

**Comment by adithya-menon-r** [comment]
Hey @IAmRiteshKoushik, is this issue under AMWOC? If yes, I would like to work on this issue.

**Comment by IAmRiteshKoushik** [comment]
@FirefoxSRV @mdxaasil @VishalTheHuman @Ashrockzzz2003

**Comment by Ashrockzzz2003** [comment]
Assigned @adithya-menon-r

**Comment by adithya-menon-r** [comment]
@Ashrockzzz2003, @IAmRiteshKoushik When is the deadline for a PR?

**Comment by Ashrockzzz2003** [comment]
You can take until `Jan 2, 2025`.

**Comment by Ashrockzzz2003** [comment]
@adithya-menon-r 
Deadline is today. If you don't reply to this issue/raise a PR, it'll be unassigned tomorrow.

**Comment by adithya-menon-r** [comment]
@Ashrockzzz2003 Yes, yes... I am almost done with the implementation... I'll create a PR by today or tomorrow morning at the latest...

**Comment by Ashrockzzz2003** [comment]
Done done. You can take another week!

**Comment by adithya-menon-r** [comment]
@Ashrockzzz2003 Ah ok... Thanks!
